fake_server.js

```diff
@@ -33,18 +33,28 @@ onmessage = (messageEvent) => {
   port.postMessage(null);
 };
 
-onfetch = (fetchEvent) => {
-  if (!new URL(fetchEvent.request.url).hostname.endsWith(".test")) {
+onfetch = async (fetchEvent) => {
+  const { url, method, headers, credentials } = fetchEvent.request;
```

Comment:
I don't think we ever need credentials?

Reply:
No, but we probably do want to assert in tests that they're definitely not there.